apps/dev-tools/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
i don't feel strongly on this, but can the root `.gitignore` just apply the ignore rule to subdirectories?

Reply:
(not blocking)

Reply:
It already probably ignores a bunch of this generic stuff, let me see if I can clean it up (it's the base `.gitignore` from creating a new vite app).

Reply:
Could completely remove it, nice catch 🙌